package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "styled-jsx-plugin-postcss",
-  "version": "3.0.2",
+  "version": "4.0.0",
   "description": "Plugin to add PostCSS support to styled-jsx",
   "main": "index.js",
   "scripts": {
-    "test": "mocha test.js"
+    "test": "mocha --timeout 5000 test.js"
```

Comment:
why this timeout?

Reply:
The default timeout of mocha is 2 seconds which was sometimes too short.